packages/core/src/components/DataTable/helpers/indexData.ts

```diff
@@ -0,0 +1,36 @@
+import memoize from 'lodash/memoize';
+
+import { ChildRow, IndexedChildRow, IndexedParentRow, ParentRow } from '../types';
+
+// eslint-disable-next-line import/prefer-default-export
+export const indexData = memoize(
```

Comment:
Why isnt default being used?

Reply:
Since it's memorized it needs to be exported as a const instead of a function, which is invalid:
https://stackoverflow.com/questions/36261225/why-is-export-default-const-invalid